src/main/java/seedu/task/commons/exceptions/UndoableException.java

```diff
@@ -0,0 +1,13 @@
+package seedu.task.commons.exceptions;
+
+/**
+ * Signals that there is no command can be undone.
+ * @author xuchen
+ *
+ */
+public class UndoableException extends Exception {
+	
+	public UndoableException() {
+		super("There is no more operation can be undone");
```

Comment:
Can change to "No more operations to undo." or "There are no more operations which can be undone"